.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: NodeJS with Webpack
```

Comment:
@mattdavis0351 mind checking this for me? I'm a little out of my league w/ workflows

Reply:
@Sterahi This *should* work.  Getting workflows to run when a pull request is merged requires digging into the webhook payload and filtering the run based on that.  There isn't a more direct way.

I also removed the `push:main` trigger for this workflow.  We should **never** be pushing directly to `main` and if we don't have branch protections in place to prevent that, at least this hurdle of the workflow not running when that happens can act as a deterrent towards people doing that.

If for some reason you need this to run on anything *other* than a successfully merged pull request, let me know, I'll add those triggers in, since again the merged part causes for a weird set of filters that are needed.

For context, a merged pull request is technically the same as a closed pull request.  They both trigger when a PR is closed, but one has a data property set to say it was merged and the other doesn't.